docs/tenancy/1.x/database-drivers.md

```diff
@@ -8,10 +8,25 @@ tags:
     - tenant
     - driver
 ---
+# Database Drivers
+
+A Database Driver is responsible for the actual management (Creation, Updating, Deletion) of the Tenant's database and/or database user. Most of the Database Drivers will run specific "elevated permissions" queries in order to provide a database and/or a database user.
+
 The principles of the Database Drivers are quite simple. The `DatabaseResolver` will fire a simple set of events that allow the database drivers to hook into. They will provide a class that implements `ProvidesDatabase`.
 
-Yes, you can install multiple database drivers, thus allowing you to use a different
-connection per [tenant](what-is-a-tenant) based on your requirements.
+Yes, you can install multiple database drivers, thus allowing you to use a different connection per [tenant](what-is-a-tenant) based on your requirements.
+
+## Available Drivers
+
+There are some Database Drivers that are maintained by the Tenancy organization, some might also be maintained by our Community.
+
+Each Database Driver will have their own installation guide.
+
+### First Party Drivers
 
 - [MySQL](database-mysql)
 - [SQLite](database-sqlite)
+
+### Second Party Drivers
+
+Check back soon!
```

Comment:
Why not third party? And not sure I would want to list these in the docs for now.